locales/te_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -5244,18 +5250,20 @@ msgstr ""
 msgid "What is this?"
 msgstr "ఇది ఏమిటి?"
 
-#  Marked fuzzy because of translated trademark
+# Marked fuzzy because of translated trademark
 #, fuzzy
```

Comment:
We should be able to remove the fuzzy marking now. I'll try to commit and revert if tests fail.

```suggestion
```